### PR TITLE
Standalone config fixed. Small bugfixes of error reporting from repos…

### PR DESCRIPTION
**Pull request by marcinczeczko**
---

I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

File: knotx-core/knotx-knot-action/src/main/resources/logback.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <layout class="ch.qos.logback.classic.PatternLayout">
+      <Pattern>
+        %d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger{36} - %msg%n
+      </Pattern>
+    </layout>
+  </appender>
+
+  <logger name="com.cognifide" level="debug"
```
Comment (tomaszmichalak) [inline knotx-core/knotx-knot-action/src/main/resources/logback.xml]: Please specify package com.cognifide.knot.action.

File: knotx-core/knotx-knot-action/src/test/resources/logback-test.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <layout class="ch.qos.logback.classic.PatternLayout">
+      <Pattern>
+        %d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger{36} - %msg%n
+      </Pattern>
+    </layout>
+  </appender>
+
+  <logger name="com.cognifide.knotx.knot.view" level="TRACE"
```
Comment (tomaszmichalak) [inline knotx-core/knotx-knot-action/src/test/resources/logback-test.xml]: knot.action.

File: knotx-core/knotx-server/src/main/java/com/cognifide/knotx/server/KnotxRepositoryHandler.java [hunk 3]
```diff
@@ -79,6 +81,14 @@ public void handle(RoutingContext context) {
     }
   }
 
+  private boolean isSuccessResponse(ClientResponse repoResponse) {
+    return repoResponse.statusCode().equals(HttpResponseStatus.OK);
```
Comment (tomaszmichalak) [inline knotx-core/knotx-server/src/main/java/com/cognifide/knotx/server/KnotxRepositoryHandler.java]: maybe HttpResponseStatus.OK.equals(repoResponse.statusCode()) - prevent NPE

File: knotx-core/knotx-server/src/main/java/com/cognifide/knotx/server/KnotxRepositoryHandler.java [hunk 4]
```diff
@@ -79,6 +81,14 @@ public void handle(RoutingContext context) {
...
+  }
+
+  private boolean isErrorResponse(ClientResponse repoResponse) {
+    return repoResponse.statusCode().equals(HttpResponseStatus.INTERNAL_SERVER_ERROR) || repoResponse.statusCode().equals(HttpResponseStatus.NOT_FOUND);
```
Comment (tomaszmichalak) [inline knotx-core/knotx-server/src/main/java/com/cognifide/knotx/server/KnotxRepositoryHandler.java]: The same.

File: knotx-core/knotx-standalone/src/main/resources/knotx-standalone.json [hunk 5]
```diff
@@ -41,7 +42,9 @@
           "x-original-requested-uri",
           "x-solr-core-key",
```
Comment (tomaszmichalak) [inline knotx-core/knotx-standalone/src/main/resources/knotx-standalone.json]: This header can be removed.

File: knotx-core/knotx-standalone/src/main/resources/knotx-standalone.json [hunk 6]
```diff
@@ -144,10 +136,40 @@
...
+              "Set-Cookie",
+              "Location"
+            ]
+          }
+        ],
+        "formIdentifierName": "_frmId"
       }
     },
-    "com.cognifide.knotx.adapter.service.http.HttpServiceAdapterVerticle": {
+    "com.cognifide.knotx.adapter.service.http.HttpServiceAdapterVerticle" :{
```
Comment (tomaszmichalak) [inline knotx-core/knotx-standalone/src/main/resources/knotx-standalone.json]: Formatting.

File: knotx-core/knotx-knot-action/src/main/resources/logback.xml (hunk 1 above)
Comment (marcinczeczko) [inline knotx-core/knotx-knot-action/src/main/resources/logback.xml]: fixed

File: knotx-core/knotx-knot-action/src/test/resources/logback-test.xml (hunk 2 above)
Comment (marcinczeczko) [inline knotx-core/knotx-knot-action/src/test/resources/logback-test.xml]: fixed

File: knotx-core/knotx-server/src/main/java/com/cognifide/knotx/server/KnotxRepositoryHandler.java (hunk 3 above)
Comment (marcinczeczko) [inline knotx-core/knotx-server/src/main/java/com/cognifide/knotx/server/KnotxRepositoryHandler.java]: fixed

File: knotx-core/knotx-server/src/main/java/com/cognifide/knotx/server/KnotxRepositoryHandler.java (hunk 4 above)
Comment (marcinczeczko) [inline knotx-core/knotx-server/src/main/java/com/cognifide/knotx/server/KnotxRepositoryHandler.java]: fixed

File: knotx-core/knotx-standalone/src/main/resources/knotx-standalone.json (hunk 5 above)
Comment (marcinczeczko) [inline knotx-core/knotx-standalone/src/main/resources/knotx-standalone.json]: fixed

File: knotx-core/knotx-standalone/src/main/resources/knotx-standalone.json (hunk 6 above)
Comment (marcinczeczko) [inline knotx-core/knotx-standalone/src/main/resources/knotx-standalone.json]: fixed

**Comment by tomaszmichalak** [comment]
Approved.

**Pull request merged**